CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 ### Fixes
 
+
+## v1.30.0 (2023-05-03)
```

Comment:
Maybe 1.29.1 if only one fix

Reply:
Agree, changed in [d74a407](https://github.com/jaegertracing/jaeger-ui/pull/1387/commits/d74a407846fd31089ffc1471e752e4acdb00ce3c) and [77498b3](https://github.com/jaegertracing/jaeger-ui/pull/1387/commits/77498b3b1ca27d267a9c8079843d2b3c74cdc4c2).